dogfooding/lib/core/repos/token_service.dart

```diff
@@ -4,6 +4,38 @@ import 'dart:convert';
 // 📦 Package imports:
 import 'package:http/http.dart' as http;
 
+enum Environment {
+  pronto(
+    'Pronto',
+    aliases: ['stream-calls-dogfood'],
+  ),
+  demo(
+    'Demo',
+    aliases: [''],
```

Comment:
Is this needed if so don't we have to put it in `staging` as well?

Reply:
Maybe it should be called subdomains instead of aliases. For demo env it just says that getstream.io without subdomain will map to it

Reply:
Then why don't we put `aliases: ['staging']` is the `Environment.staging` constructor?

Reply:
oh yes thats why I named it alias :D because I'm matching the env when subdomain equals to the name of the env (demo, pronto, staging) or matches one of the aliases. So `statging` is not needed because its the name